index.md

```diff
@@ -23,6 +23,15 @@ many other activities:
 ::::{grid} 1 2 2 3
 :gutter: 3
 
+:::{grid-item-card} PyMC 4.0 is ready
+:link: v4_announcement
+:link-type: ref
+:class-header: bg-pymc-one
```

Comment:
I would check the color and in general update all the cards. Given the importance of this announcement we could make it fill a whole row too for example. I think we can remove the gsoc and the welcome to the blog one

Reply:
How do we make it full a whole row? I'm very confused by the `:::`.